platform/jvm/BUILD

```diff
@@ -46,3 +46,10 @@ java_binary(
     main_class = "com.vaticle.bazel.distribution.platform.jvm.MainKt",
     visibility = ["//visibility:public"],
 )
+
+config_setting(
```

Comment:
During the review of the previous PR I didn't see this being used anywhere, but it turns out TypeDB Studio uses it.